commit 9c682bac219e28afc73e25e7089b553ecd1ca0f6
Author: Keno Fischer <keno@juliacomputing.com>
Date:   Wed Sep 30 22:15:40 2020 -0400

    [clang/CMake] Respect LLVM_TOOLS_INSTALL_DIR
    
    Otherwise clang installs all of its tools into `bin/` while
    LLVM installs its tools into (LLVM_TOOLS_INSTALL_DIR).
    I could swear this used to work (and in fact the julia build system
    assumes it), but I can't pin down a specific commit that would
    have broken this, and julia has been relying on pre-compiled binaries
    for a while now (that don't use this setting), so it may have been
    broken for quite a while.
    
    Differential Revision: https://reviews.llvm.org/D88630

diff --git a/tools/clang/cmake/modules/AddClang.cmake b/tools/clang/cmake/modules/AddClang.cmake
index 704278a0e93..a455365cabf 100644
--- a/tools/clang/cmake/modules/AddClang.cmake
+++ b/tools/clang/cmake/modules/AddClang.cmake
@@ -170,7 +170,7 @@ macro(add_clang_tool name)
 
     install(TARGETS ${name}
       ${export_to_clangtargets}
-      RUNTIME DESTINATION bin
+      RUNTIME DESTINATION ${LLVM_TOOLS_INSTALL_DIR}
       COMPONENT ${name})
 
     if(NOT LLVM_ENABLE_IDE)
